examples/6field-simple/data/BOUT.inp

```diff
@@ -229,24 +229,17 @@ hyperviscos = -1.0  # Radial hyper viscosity
 phi_curv = true    # Include curvature*Grad(phi) in P equation
 # gamma = 1.6666
 
-## field inversion flags: Add the following
-#  1 - Zero-gradient DC component on inner boundary
-#  2 - Zero-gradient AC component on inner boundary
-#  4 -      "        DC     "      " outer    "
-#  8 -      "        AC     "      " outer    "
-# 16 - Zero all DC components of the result
-# 32 - Don't use previous solution to start iterations
-#      (iterative methods only) 
-# 64 - Set the width of the boundary layer to 1
-# 128 - use 4th order differencing
-# 256 - Laplacian = 0 inner boundary (combine 2nd & 4th-order)
-# 512 - Laplacian = 0 outer boundary ( sometimes works )
-
-#phi_flags = 74  # inversion flags for phi (2+8+64+128)
-phi_flags = 769  # 256 + 512
-
-#apar_flags = 74 # 2+8
-apar_flags = 769
+[phiSolver]
+#inner_boundary_flags = 1 + 2 + 128 # INVERT_DC_GRAD + INVERT_AC_GRAD + INVERT_BNDRY_ONE
+#outer_boundary_flags = 1 + 2 + 128 # INVERT_DC_GRAD + INVERT_AC_GRAD + INVERT_BNDRY_ONE
+inner_boundary_flags = 1 + 4 # INVERT_DC_GRAD + INVERT_AC_LAP
+outer_boundary_flags = 1 + 4 # INVERT_DC_GRAD + INVERT_AC_LAP
+
+[aparSolver]
+#inner_boundary_flags = 1 + 2 + 128 # INVERT_DC_GRAD + INVERT_AC_GRAD + INVERT_BNDRY_ONE
+#outer_boundary_flags = 1 + 2 + 128 # INVERT_DC_GRAD + INVERT_AC_GRAD + INVERT_BNDRY_ONE
+inner_boundary_flags = 1 + 4 # INVERT_DC_GRAD + INVERT_AC_LAP
+outer_boundary_flags = 1 + 4 # INVERT_DC_GRAD + INVERT_AC_LAP
```

Comment:
For 5.0, let's work out a way to use the names instead of numerical flags for this